beyond-essence/inventory/cards.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: beyond-essence
+header_image: private/main.jpg
+title: Cards
+background_image: index.jpg
+---
+
+Cards are visual elements that are used to group relating pieces of information into one compact and manageable unit.
+
+## Use cases
+
+Cards are used to display various information on a superordinate topic.
+
+In the Beyond cockpit we provide two types of cards:
+
+* Standard cards
+
+{% image_custom image="/assets/img/pages/essence/cards-standard.png"  width="100" lightbox %}
+
+* Cards with title
+
+{% image_custom image="/assets/img/pages/essence/cards-title.png"  width="100" lightbox %}
+
```

Comment:
We also use cards for topic clustering: 
<img width="1013" alt="Bildschirmfoto 2019-08-13 um 10 24 13" src="https://user-images.githubusercontent.com/6104802/62926450-d661b000-bdb4-11e9-84a6-347edc01660b.png">

These cards on this page are special and differ from the others e.g.  in relation to the amount of text and UI elements